### PR TITLE
Fix potential error with wallets overview while wallets are loading

### LINKED ISSUE #1073: dictionary changed size during iteration
On startup after upgrading to 1.3.0, specter shows:

Details:
Traceback (most recent call last):
  File "flask/app.py", line 1950, in full_dispatch_request
  File "flask/app.py", line 1936, in dispatch_request
  File "flask_login/utils.py", line 272, in decorated_view
  File "cryptoadvance/specter/server_endpoints/wallets.py", line 76, in wallets_overview
RuntimeError: dictionary changed size during iteration

Clicking on the button brings me into a seemingly flawless specter environment.

### PR DESCRIPTION
**Pull request by ben-kaufman**
Fix #1073

### CODE REVIEW

File: src/cryptoadvance/specter/server_endpoints/wallets.py [hunk 1]
```diff
@@ -73,7 +73,7 @@ def wrapper(*args, **kwargs):
 @login_required
 def wallets_overview():
     app.specter.check_blockheight()
-    for wallet in app.specter.wallet_manager.wallets.values():
+    for wallet in list(app.specter.wallet_manager.wallets.values()):
```
Comment (djpnewton) [inline src/cryptoadvance/specter/server_endpoints/wallets.py:76]: why is this needed? wallet_manager.wallets is a dict (https://github.com/cryptoadvance/specter-desktop/blob/2baa83f572ed59e8d41ced4bc8792cc739d47998/src/cryptoadvance/specter/wallet_manager.py#L54) so shouldnt `values()` always return a valid list?

Comment (ben-kaufman) [reply src/cryptoadvance/specter/server_endpoints/wallets.py:76]: Yes it is a valid list, but since wallets are being loaded asynchronously, the length of the list could change during iteration, which will cause an error. This just clones the list so it won't be affected by async changes in the dictionary.

**Comment by k9ert** [comment]
tested, LGTM. Ready to merge.
